deploy/operator.yaml

```diff
@@ -67,4 +72,5 @@ metadata:
   name: nmstate-config
   namespace: nmstate
 data:
-  node_network_state_refresh_interval: "5"
\ No newline at end of file
+  node_network_state_refresh_interval: "5"
+  interfaces_filter: "veth*"
```

Comment:
related to the test of two interfaces should this be `"{veth*,}"`

Reply:
and maybe we should update the documentation about this option

Reply:
We should absolutely document this in https://github.com/nmstate/kubernetes-nmstate/blob/master/docs/developer-guide-state.md. @HlinaCZ could you add one paragraph there, mentioning that we have this config map with the filter value, it is glob, show examples you used in test.

Reply:
I thought this file https://github.com/nmstate/kubernetes-nmstate/blob/master/docs/user-guide-state-reporting.md would be more appropriate for its use case.